addon/index.js

```diff
@@ -36,6 +35,7 @@ const SERVER_ENVIRONMENTS = {
 
 const store = require('sdk/simple-storage').storage;
 const {Cc, Ci, Cu} = require('chrome');
+Cu.import('resource://gre/modules/Services.jsm');
```

Comment:
Re: ESLint errors,

```
   88:5  error    "Services" is not defined             no-undef
   91:5  error    "Services" is not defined             no-undef
  210:5  error    "Services" is not defined             no-undef
```

We probably just need to throw something like this at the top of the file:
```js
/* global Services:false */
```